src/Groups/Groups/examples/v1.0-beta/New-MgGroupLifecyclePolicy.md

```diff
@@ -1,12 +1,9 @@
-### Example 1: Using the New-MgGroupLifecyclePolicy Cmdlet
-```powershell
-Import-Module Microsoft.Graph.Groups
-$params = @{
-	GroupLifetimeInDays = 100
-	ManagedGroupTypes = "Selected"
-	AlternateNotificationEmails = "admin@contoso.com"
-}
-New-MgGroupLifecyclePolicy -BodyParameter $params
-```
-This example shows how to use the New-MgGroupLifecyclePolicy Cmdlet.
-To learn about permissions for this resource, see the [permissions reference](/graph/permissions-reference).
+### Example 1: Code snippet
+
+```powershell
Import-Module Microsoft.Graph.Groups
+
+Get-MgGroupLifecyclePolicy

```

Comment:
Cmdlet mismatch - `New-MgGroupLifecyclePolicy` vs `Get-MgGroupLifecyclePolicy`.

Reply:
Thanks Peter for flagging this. I am definitely reviewing the logic